### ISSUE TITLE
Navigation bar between different formats

### ISSUE THREAD

**Issue by Saborknight** [Saborknight/9liner-and-notepad#1]
Implement a navigation bar for navigating between the different types of pages.

> What do you think about adding additional buttons to the notepad, which the player can use to navigate through the diffrent sheets (nineline...) ? The only reason I'm not using this mod, are the uncountable keybindings, which also interfere with mods like TFAR.
> I would like to see a version, which requires one keybind(to open the notepad), and everything else can be opened from the dialogue itself.

**Submitted by** [R3vo](https://forums.bistudio.com/profile/932184-r3vo/)

## Forum Responses

> That's a good suggestion! I think i'll add a little navigationbar at the bottom of each document.

**Submitted by** [chief_wiggum](https://forums.bistudio.com/profile/926664-chief_wiggum/)

**Comment by TheMysteriousVincent** [comment]
ACE interaction menu solves this problem, I guess.

**Comment by Saborknight** [comment]
Not if we want to maintain the only dependency as being CBA.. Relying on the ACE interaction menu would mean people not using ACE (as weird as they may be) would not be able to navigate between sheets.

**Comment by TheMysteriousVincent** [comment]
Well then we have to remove the `ace_interaction_menu` dependency in CfgPatches in `nln_main`.
Do you mean something like that?:
![20171219205446_1](https://user-images.githubusercontent.com/10392626/34176739-5be8d246-e501-11e7-95ce-59826c8d0e4a.jpg)

**Comment by Saborknight** [comment]
Nope, but that is waaaaay better than what I was thinking about xD

Need to incorporate these to the other sheet formats in their own styles.

**Comment by Saborknight** [comment]
There's a slight problem with the suggested navigation bar in the image above Though. It's not easily scalable unless we add sideways "scroll" (using clicks on an arrow in the desired direction) buttons, or something similar.

Given that a navigation menu has been suggested, is this still necessary? It does offer a second method of navigation which is intuitive and requires less clicks when all the user wants to do is change sheet format.
I think having this feature alongside the main menu would be a good idea. However, in the interest of getting a new version released sooner, I'll stick this in v1.1.

**Comment by Saborknight** [comment]
See #50 for details on the Main Menu dialogue feature

**Comment by Saborknight** [comment]
In addition to this, we should find a new place to indicate to the user what page they're on. Having a hint tell you is conflicting with other notices, such as the "Page %1 saved" notices. They're fighting to be noticed.
